images/installer/Dockerfile.upi.ci

```diff
@@ -58,8 +58,10 @@ RUN curl -sSL "${ALIYUN_URI}" --output /tmp/aliyun-cli-linux-latest-amd64.tgz &&
 RUN curl https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip.py
 RUN python get-pip.py 'pip<21.0'
 RUN python -m pip install pyopenssl
-ENV CLOUDSDK_PYTHON=/usr/bin/python
+ENV CLOUDSDK_PYTHON=/usr/bin/python3
 
+RUN pip3 install --upgrade pip
+RUN python3 -m pip install pyOpenSSL
```

Comment:
From https://github.com/openshift/release/pull/38668, seem like only need to export CLOUDSDK_PYTHON to python3, are these lines are necessary?

Reply:
FYI Just retried without these 2 lines, there will be error "The signurl command requires the pyopenssl library (try pip install pyopenssl or easy_install pyopenssl)" (see [job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/7118/pull-ci-openshift-installer-master-e2e-gcp-ovn-upi/1650691196860764160)).

Reply:
good to know, seem like when running gcloud command against storage bucket, need the library. Thanks for the fix.